homeassistant/components/blink/services.yaml

```diff
@@ -2,50 +2,36 @@
 
 blink_update:
   fields:
-    device_id:
+    config_entry_id:
```

Comment:
Since this service is deprecated, I suggest we don't change the interface now. Keep it as is and then we just remove it in six months.

Reply:
I broke the interface in a previous change moving the services https://github.com/home-assistant/core/pull/103435 - I think I should at least fix it.  Or I could move it back to the original code (but it was not in the right spot nor right).

Reply:
Is something not working currently? The previous change has already been released. I suggest we don't make an additional breaking change here.